src/mvAppItem.h

```diff
@@ -314,26 +314,28 @@ GetEntityCommand(mvAppItemType type)
     case mvAppItemType::mvInputFloatMulti:             return "add_input_floatx";
     case mvAppItemType::mvInputDoubleMulti:            return "add_input_doublex";
     case mvAppItemType::mvDragPoint:                   return "add_drag_point";
+    case mvAppItemType::mvDragRect:                    return "add_drag_rect";
     case mvAppItemType::mvDragLine:                    return "add_drag_line";
     case mvAppItemType::mvAnnotation:                  return "add_plot_annotation";
+    case mvAppItemType::mvTag:                         return "add_plot_tag";
     case mvAppItemType::mvLineSeries:                  return "add_line_series";
     case mvAppItemType::mvScatterSeries:               return "add_scatter_series";
     case mvAppItemType::mvStemSeries:                  return "add_stem_series";
     case mvAppItemType::mvStairSeries:                 return "add_stair_series";
     case mvAppItemType::mvBarSeries:                   return "add_bar_series";
+    case mvAppItemType::mvGroupBarSeries:              return "add_group_bar_series";
     case mvAppItemType::mvErrorSeries:                 return "add_error_series";
     case mvAppItemType::mvCustomSeries:                return "add_custom_series";
-    case mvAppItemType::mvVLineSeries:                 return "add_vline_series";
```

Comment:
`add_hline_series` and `add_vline_series`: We definitely _can_ keep both these functions by adding them to `_deprecated.py`, telling the user to use `add_inf_line_series` instead, and redirecting the call to `add_inf_line_series` (with `horizontal=True` if needed). We only need a thin Python wrapper in `_deprecated.py`, no C++ code involved.